### ISSUE TITLE
Getting hammered with errors

### ISSUE THREAD

**Issue by sibraxlis** [PikaMug/Quests#1188]
**What is your Quests version / build number (do _not_ say "latest")?**
3.9.3-b96
**Describe the problem and how to replicate it. Any console errors to post _using Pastebin.com_?**
https://pastebin.com/Jr0tK3Pi

I also get player interact errors.

**Comment by PikaMug** [comment]
Looks as if DungeonsXL might've changed some important code. I'll look into it!

**Comment by sibraxlis** [comment]
Removing the dungeonsXL module did not fix it, nor did updating to b99 like some others did.

**Comment by PikaMug** [comment]
@sibraxlis As far as I can tell, the crucial part of your log is:

```
Caused by: java.lang.NoSuchMethodError:
de.erethon.dungeonsxl.player.DGroup.getByPlayer(Lorg/bukkit/entity/Player;)Lde/erethon/dungeonsxl/player/DGroup;
    at me.blackvein.quests.Quester.getMultiplayerQuesters(Quester.java:3751) ~[Quests-3.9.3.jar:?]
```

This indicates a problem linking with DungeonsXL itself (for party/group support) and _not_ the module.

**Comment by PikaMug** [comment]
Addressed in 311eac2d2858a6e9d49eae703897982195aa37dd

@sibraxlis Use most recent development build:
https://ci.codemc.io/job/PikaMug/job/Quests/

You'll also need to use this updated module (not yet released on Spigot):
https://www.dropbox.com/s/09dn1n02218hlaz/DungeonsXLQuestsModule-1.2.jar?dl=0
